<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntityEditor</class>
 <widget class="QWidget" name="EntityEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="topHLayout">
     <item>
      <widget class="QLabel" name="selectedComponentsLabel">
       <property name="text">
        <string>Selected Components</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="availableComponentsLabel">
       <property name="text">
        <string>Available Components</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="selectionHLayout">
     <item>
      <widget class="QListWidget" name="selectedComponentsListWidget"/>
     </item>
     <item>
      <widget class="QListWidget" name="availableComponentsListWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionsHLayout">
     <item>
      <spacer name="leftHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="deselectPushButton">
       <property name="text">
        <string>➡️</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectPushButton">
       <property name="text">
        <string>⬅️</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="rightHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
